qiskit/algorithms/__init__.py

```diff
@@ -72,12 +72,14 @@
    MaximumLikelihoodAmplitudeEstimationResult
 
 
-Eigensolvers
-------------
+Eigen Solvers
+-------------
 
 Algorithms to find eigenvalues of an operator. For chemistry these can be used to find excited
 states of a molecule, and qiskit-nature has some algorithms that leverage chemistry specific
-knowledge to do this in that application domain.
+knowledge to do this in that application domain.These algorithms are pending depreciation.
```

Comment:
```suggestion
knowledge to do this in that application domain. These algorithms are pending depreciation.
```

Reply:
I also realized it says depreciation instead of deprecation 🤣. Changing both egeinsolvers and numpy eigensolvers

Reply:
>depreciation
Well they are losing their value as such! 🤣